src/bionics.cpp

```diff
@@ -595,6 +595,14 @@ bool player::activate_bionic( int b, bool eff_only )
             tank_plut = 0;
             reactor_plut = 0;
             }
+    } else if (bio.id == "bio_cable") {
+        bool has_cable = has_item_with( []( const item &it ) {
+            return it.has_flag( "CABLE_SPOOL" ) && it.active;
```

Comment:
`it.active` is much cheaper. I'd put it before `it.has_flag( "CABLE_SPOOL" )` (but it's not a big deal).